### PR TITLE
Node5 merge

### PR DESCRIPTION
**Pull request by doug-martin**
- [ ] @dustinsmith1024

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+sudo: false
+
+after_script:
+  - grunt coveralls
+
 language: node_js
 node_js:
-  - "0.10"
\ No newline at end of file
+  - "4.0"
```
Comment (dustinsmith1024) [inline .travis.yml]: i only tested on 5.1.  Did you check if 4 works okay?

Comment (dustinsmith1024) [inline .travis.yml]: actually it looks like you set engine >=5 so i think we could remove this.

File: package.json [hunk 2]
```diff
@@ -1,25 +1,44 @@
...
-    },
-    "scripts": {
-        "test": "export NODE_PATH=lib:$(NODE_PATH) && export NODE_ENV=test && ./node_modules/it/bin/it"
-    },
-    "engines": {
-        "node": ">= 0.6.1"
-    }
+  "name": "comb",
+  "description": "A framework for node",
+  "version": "0.4.0",
```
Comment (dustinsmith1024) [inline package.json]: should we up to 0.5.0 or 1.0.0??

File: lib/base/misc.js [hunk 3]
```diff
@@ -41,7 +42,7 @@ function argsToArray(args, slice) {
  */
 function isBoolean(obj) {
     var undef, type = typeof obj;
-    return obj != undef && type == "boolean" || type == "Boolean";
```
Comment (dustinsmith1024) [inline lib/base/misc.js]: just making sure you wanted all these to change to =='s??

File: .travis.yml (hunk 1 above)
Comment (doug-martin) [inline .travis.yml]: Good catch on the engine...I have travis testing both right now so Ill change the engine to include the stable version also.

**Pull request closed without merging**